src/coreComponents/functions/TableFunction.cpp

```diff
@@ -83,7 +83,7 @@ void TableFunction::readFile( string const & filename, array1d< real64 > & targe
   }
   catch( std::runtime_error const & e )
   {
-    GEOS_THROW( GEOS_FMT( "{} {}: {}", catalogName(), getDataContext(), e.what() ), InputError );
+    GEOS_THROW( GEOS_FMT( "{}: {}", catalogName(), e.what() ), InputError, getDataContext()  );
```

Comment:
```suggestion
    GEOS_THROW( e.what(), InputError, getDataContext()  );
```

andsame below for `catalogName()` deletion.

Reply:
and below and below and below and below and below and below